app/views/stats/range.html.erb

```diff
@@ -1,7 +1,53 @@
 <h1>Activity</h1>
 
-<p>From <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %></p>
-
+<h5>From <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %></h5>
+<h4><b> Select a range below to view stats within the range </b></h4>
+<%= form_tag request.url, method: 'get' do %>
+    <div class="row">
+    <div class="col-md-8" style="background:#F5F6F8;">
+        <p>
+            <button data-placement="bottom" data-toggle="popover" data-container="body" data-placement="bottom" type="button" data-html="true" class="btn btn-primary">
+                <span><i class="fa fa-calendar fa-2x"> </i> Today </span> </button>
+             </button>   Compared to  <%= Date.today.to_formatted_s(:long) %>
```

Comment:
Ah, here maybe there is a 2nd redundant `button` element causing trouble? Also, is the `span` totally necessary? It might be causing some layout issues too... just not totally sure it's required. Thanks!